crates/extensions/c8y_http_proxy/src/messages.rs

```diff
@@ -64,6 +68,8 @@ pub type EventId = String;
 
 pub type Unit = ();
 
+pub type Bool = bool;
+
```

Comment:
Is this still used? I don't think so as the call to check the end-point of an url has been removed;